andrewinsoul/Store-Manager
# user signup feature

#### What does this PR do?
This PR creates the signup screen of the app and also adds user who signed up successfully to the firebase service

#### Description of Task to be completed?
- Create screen for the signup feature
- Link the signup from the screen to the firebase service

#### How should this be manually tested?
- Fetch branch locally
- Run npm install to install all dependencies
- run `./node_modules/.bin/react-native run-android` OR `./node_modules/.bin/react-native run-ios` depending on your device/simulator
- You should see the signup screen of the app, now fill the details and signup. Your details including the token will be logged to your console


#### Any background context you want to provide?
- Syncing the value on the entry field to the state assuming the user pastes a value into the entry field. Currently, the state does not align with the entry field when the user just pastes the value in the entry field. 

#### What are the relevant pivotal tracker stories?
N/A

#### Screenshots (if appropriate)
N/A
